### DISCUSSION TITLE
Quarto with R language in Vscode not run

### DISCUSSION THREAD

**Discussion by DrDinhLuong** [quarto-dev/quarto-cli#9629] [Q&A]
### Description

I using R language with Quarto in Vscode. However, I have a problems when push "run cell" it not working.

<img width="1315" alt="Screenshot 2024-05-09 at 18 17 19" src="https://github.com/quarto-dev/quarto-cli/assets/86107710/d08c62c0-70c2-489e-b742-3e3267e5a814">
Pls, help me.

**Comment by mcanouil** [comment]
Quarto does not setup your R/Python/Julia environment, it only uses it.
You need to install the R packages you need and more generally setup yourself your environment.
There are plenty of online resources dedicated to that.

**Reply by DrDinhLuong** [reply]
Actually, I already installed this package. So I don't know why it not working when I call it

**Reply by mcanouil** [reply]
Here the error is unequivocal, you do not have `ggplot2` installed in your environment.
You need to check your R environment.
I'm afraid it's unrelated to Quarto CLI or the Quarto VSCode extension.
